src/financial-assistance.md

```diff
@@ -3,32 +3,52 @@ layout: default
 title: Financial Assistance
 ---
 
-Our financial assistance programme exists to support attendees who might otherwise find it difficult to attend the conference.
-We have a budget set aside for this purpose.
-
-The deadline for applications is {{ site.con_financial_assistance_deadline }} {{ site.con_year }}<br />
-The organisers will not be able to accept applications after that date.
-
-The financial assistance programme prioritises speakers.
-We also use it to support attendees from groups who are under-represented in our community and at our events,
-as part of our efforts to make PyCon UK as inclusive and diverse as possible.
-
-Sometimes people feel reluctant to apply,
-for all kinds of reasons.
-For example, they feel might embarrassed to ask for help,
-or they doubt that they are the right kind of person,
-or they think that they don't deserve it,
-or that other people are more deserving.
-We hope that you won't feel like this, and we will be glad to receive your request for support.
-
-We welcome applications from all attendees, and strongly encourage you to apply if it would make it easier for you to attend.
-Many of your fellow attendees will be doing the same, including many speakers and even members of the conference organisation team.
-
-We'd love to be able to help all those who submit applications, but our funds are not unlimited, so we can't meet all requests.
-We do our best to maximise the impact of our financial assistance funding.
-Whoever you are, however, and whatever your needs, please do apply if we might be able to help you attend.
+PyCon UK is pleased to offer grants for those who might otherwise not be able to attend.
+
+## Contributions
+You can help by purchasing a <strong>Supporter</strong> ticket which will allow us to allocate extra funds to providing cheaper or free tickets to those who would not otherwise be able to afford to attend.
+
+## Eligibility
+We encourage anyone who needs assistance to attend to apply for a grant. We’d love to be able to help all those who submit applications, but our funds are not unlimited, so we can’t meet all requests. Our goal is to maximise the impact our grant program has, within the limited funds that we have available.
+
 Your application will be treated in complete confidence.
 
-Thanks for applying. We hope we'll be able to help.
+Our selection process will take into account:
+
+* The contribution that the applicant can make to the conference and the wider community. This may be because you are a speaker or workshop/tutorial presenter, or because you have an impact on the Python community in some other way (e.g. local community organisers, open source contributors).
```

Comment:
```suggestion
* Whether PyCon UK is the most local Python event that the applicant could possibly attend.
* The contribution that the applicant can make to the conference and the wider community. This may be because you are a speaker or workshop/tutorial presenter, or because you have an impact on the Python community in some other way (e.g. local community organisers, open source contributors).
```

Reply:
Happy for that to be either first or second, but I think we need to say something about how we're more likely to fund somebody where PyCon UK is the closest event for them above somebody who could go to some other event nearer to them.

Reply:
as per the bit in brackets in point 2 of https://ukpa-internaldocs.readthedocs.io/en/latest/policies/pyconuk-financial-assistance-policy/